windows/src/Defines.mak

```diff
@@ -28,50 +28,50 @@ INSTALLPATH_KEYMANDEVELOPER=%ProgramFiles(X86)%\Keyman\Keyman Developer
 INSTALLPATH_KEYMANENGINE=%CommonProgramFiles(X86)%\Keyman\Keyman Engine
 
 !IFDEF DEBUG
-  MAKEFLAG_DEBUG=-DDEBUG
-  DELPHI_MSBUILD_FLAG_DEBUG=/p:Config=Debug
+MAKEFLAG_DEBUG="DEBUG=$(DEBUG)"
+DELPHI_MSBUILD_FLAG_DEBUG="/p:Config=Debug"
```

Comment:
nmake does not permit whitespace in the same way as Borland make. You'll see these changes throughout.